dora/stress/common/src/main/java/alluxio/stress/worker/WorkerBenchTaskResult.java

```diff
@@ -155,11 +161,42 @@ public void setErrors(List<String> errors) {
     mErrors = errors;
   }
 
+  public List<Long> getDurationPercentiles() {
+    return mDurationPercentiles;
+  }
+
+  public void setDurationPercentiles(List<Long> percentiles) {
+    mDurationPercentiles = percentiles;
+  }
+
+  public void generatePercentiles() {
+    Histogram durationHistogram = new Histogram(
+            FormatUtils.parseTimeSize(mParameters.mDuration)
+                    + FormatUtils.parseTimeSize(mParameters.mWarmup),
```

Comment:
why do you add warm up here?

Reply:
This parameter defines the biggest possible value of the histogram. Since we haven't removed the warmup step, it is still possible for one file-read process to be longer than the entire duration (when duration is very small). But it won't be a problem to set the value as the duration, I will fix it.